jina/executors/indexers/cache.py

```diff
@@ -59,28 +65,29 @@ def close(self):
             pickle.dump(self.id_to_cache_val, open(self.path + '.ids', 'wb'))
             pickle.dump(self.cache_val_to_id, open(self.path + '.cache', 'wb'))
 
-    supported_fields = [ID_KEY, CONTENT_HASH_KEY]
-    default_field = ID_KEY
+    default_fields = [ID_KEY]
 
-    def __init__(self, index_filename: Optional[str] = None, field: Optional[str] = None, *args, **kwargs):
+    def __init__(self, index_filename: Optional[List[str]] = None, fields: Optional[List[str]] = None, *args, **kwargs):
```

Comment:
`index_filename` is a `List[str]`?

Reply:
I would like better for `fields` to be tuples?

Reply:
Yes, fixed